### PR TITLE
Update networking page to match API changes

### LINKED ISSUE #662: Update Networking page for IP endpoint changes

### PR DESCRIPTION
**Pull request by briansteffens**
Closes #662

### CODE REVIEW

**Comment by briansteffens** [comment]
screen shot 2016-12-20 at 9 57 14 am
screen shot 2016-12-20 at 9 57 58 am

**Comment by na3d** [comment]
Just curious, what is the thought behind moving the Add private IP address button?

**Comment by briansteffens** [comment]
I thought it looked weird on the other side of the IPv6 field from the part of the page it affected. Phil is saying not to have 4/6 sections under private so that may change it again more to how it was before

**Comment by na3d** [comment]
What if we reused what we did for config disks?

**Comment by briansteffens** [comment]
@ndodson: how would what we did with config disks map to this?

@ndodson, @peaton here's another possible layout:

screen shot 2016-12-20 at 10 28 13 am
screen shot 2016-12-20 at 10 28 23 am

**Comment by eatonphil** [comment]
I like that latest screenshot. +1

File: test/linodes/linode/networking/layouts/SummaryPage.spec.js [hunk 1]
```diff
@@ -15,27 +17,8 @@ describe('linodes/linode/networking/layouts/SummaryPage', () => {
     sandbox.restore();
   });
 
-  const linodes = {
```
Comment (eatonphil) [inline test/linodes/linode/networking/layouts/SummaryPage.spec.js]: Nice!!

**Comment by na3d** [comment]
The "Public network" header should probably be changed then, since there is private network information in there.

**Pull request merged**